
syntax = "proto3";

package engine; 


// The request message containing the user's name.
message TemplateRegisterRequest {
    string name = 1;
    repeated string source = 2;

    bytes source_package = 3;
}

// The response message containing the greetings
message TemplateRegisterReply {
    bool result = 1;
}

// The greeting service definition.
service EngineService {
    // Sends a greeting
    rpc RegisterTemplate (TemplateRegisterRequest) returns (TemplateRegisterReply) {}
}
